internal/common/version.go

```diff
@@ -70,3 +71,18 @@ func GetMajorMinorVersion(version string) (*string, error) {
 	versionStr := fmt.Sprintf("%s.%s", splittedVersion[0], splittedVersion[1])
 	return &versionStr, nil
 }
+
+func IsVersionPreRelease(str string) (*bool, error) {
+	str = strings.TrimSuffix(str, "-multi")
+	semVersion, err := version.NewVersion(str)
+	if err != nil {
+		return nil, err
+	}
+
+	return swag.Bool(semVersion.Prerelease() != ""), nil
+}
+
+func CheckIfValidVersion(v string) error {
```

Comment:
Why do we need to wrap this?

Reply:
For clarity reasons mainly, as to why we create a version and not use it